lms/envs/common.py

```diff
@@ -1257,7 +1257,7 @@
     'js/vendor/url.min.js',
     'common/js/vendor/underscore.js',
     'common/js/vendor/underscore.string.js',
-    'common/js/vendor/picturefill.min.js',
```

Comment:
I believe the RequireJS Optimizer takes care of all of our minification needs when it creates the production bundle.